[DX-1684] Release notes for 5.0.14 (attempt 2) (#5428)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.0.md

---

## 5.0.14 Release Notes

### Release Date 18th September 2024

> ### Release Highlights

Reviewer: ```suggestion
### Upgrade Instructions

This release is not tightly coupled with [Tyk Gateway v5.0.14]({{< ref "product-stack/tyk-gateway/release-notes/version-5.0">}}) so you do not have to upgrade both together.

### Release Highlights
```

> This release fixes some display issues in Tyk Dashboard and Tyk Classic Portal when using PostgreSQL.

Reviewer: ```suggestion
This release fixes some display issues in Tyk Dashboard and Tyk Classic Portal when using PostgreSQL.

Please note, the fixes in this patch release are unrelated to the [critical fixes of Tyk Gateway v5.0.14]({{< ref "product-stack/tyk-gateway/release-notes/version-5.0">}})
```

### Changelog {#Changelog-v5.0.14}

#### Fixed

<ul>
<li>
<details>

> <summary>Fixed display issue for API statistics</summary>

Reviewer: ```suggestion
<summary>Tyk Dashboard UI: Fixed display issue for API statistics</summary>
```

Fixed an issue where API statistics were not being shown when using PostgreSQL and adding two or more tags in the Activity page
</details>
</li>
<li>
<details>

> <summary>Fixed issue with display of HTTP 429 status codes on the Activity page</summary>

Reviewer: I understand the fixes are only in the UI not the APIs, right @andyo-tyk

> <summary>Fixed issue with display of HTTP 429 status codes on the Activity page</summary>

Reviewer: ```suggestion
<summary>Tyk Dashboard UI:  Fixed issue with display of HTTP 429 status codes on the Activity page</summary>
```

Fixed an issue where HTTP 429 status codes were not being shown on the Activity page when using PostgreSQL
</details>
</li>
<li>
<details>

> <summary>Fixed display of graphs and requests counter on Tyk Classic Portal</summary>

Reviewer: ```suggestion
<summary>Tyk Classic Portal UI: Fixed display of graphs and requests counter</summary>
```

Fixed wrong graphs and incorrect requests counter on Tyk Classic Portal when using PostgreSQL
</details>
</li>
<li>
<details>

> <summary>Fixed Error Breakdown display issues with dates</summary>

Reviewer: ```suggestion
<summary>Tyk Dashboard UI: fixed issues with the Error Breakdown display, specifically related to date handling</summary>
```
